source/styleEditor.js

```diff
@@ -1,3 +1,18 @@
 $(document).ready(function(){
-  // your code goes here.
-});
+  $('form').on('submit', function(event) {
+    event.preventDefault()
+  })
+  var submitButton  = document.getElementsByTagName('input')[3]
+  submitButton.addEventListener('click', stylePage)
+})
+
+function stylePage() {
+  var selector = $('input[name=selector]').val()
+  var property = $('input[name=property]').val()
+  var value = $('input[name=value]').val()
+  if(selector == '' || property == '' || value == '') {
```

Comment:
I know this if statement that reloads the page based on the presence of any empty input fields isn't part of the spec, but it's something that I incorporated early when I was building out this function. I kept it though because I though because I suppose I found the `location.reload()` function to be quite interesting.

Reply:
This is a pretty cool find! It would be nice to maybe provide the user an error if any of the fields were blank so they could correct it :)